app/assets/javascripts/listings/components/listing/sale-unit-stats.html.slim

```diff
@@ -25,6 +25,23 @@ div.table-pricing-wrapper
             span.ui-icon.ui-static.ui-medium.i-splash.message-icon ng-if="::priceGroup.Reserved_Type"
                 svg
                   use xlink:href="{{::$ctrl.reservedDescriptorIcon($ctrl.parent.listing, priceGroup.Reserved_Type)}}"
+          td.is-nested.table-cell-units.cell-hide-small-only  data-th="{{ 't.units' | translate }}"
```

Comment:
I'd be interested in hearing what shawn has to say about the show for small vs hide for small strategy when we talk to him tomorrow (?) if we have time.